### DISCUSSION TITLE
아이템 가격 무료 설정

### DISCUSSION THREAD

**Discussion by strangestraw** [naverz/zepeto-studio-kor#165] [5. Q&A]
안녕하세요.

제페토 월드 내에서 아이템을 무료로 판매/지급 하려고합니다.
아이템 생성 시 최소 가격이 3ZEM인데, 무료로 설정이 가능한가요?
가이드 부탁드립니다.

감사합니다.

**Comment by hyeonjin-jo** [comment]
안녕하세요.

월드 상품 만들기 진행시 0ZEM으로 세팅이 가능합니다.

아래의 링크를 참고해서 진행 부탁드립니다.

https://studio.zepeto.me/kr/guides/making-world-products
